app/components/auk/input.js

```diff
@@ -1,5 +1,12 @@
 import Component from '@glimmer/component';
 
+/**
+ * Simple wrapper for Ember's <Input> component. Takes all arguments that <Input> takes.
+ *
+ * @argument {String} icon
+ * @argument {Boolean} block
+ * @argument {Boolean} error
+ */
 export default class AuInput extends Component {
```

Comment:
`export default class AuInput extends Component {` 
niet aangepast of express zo gelaten? 
Ik zou durven denken dat class Input mogelijk conflicteert met de <Input>, maar niet zeker hiervan

Reply:
Check. Aangepast. Klasse-namen zijn eigenlijk puur "voor de show". Je zou eigenlijk evengoed anonieme klasses kunnen definiëren, maar dat wordt sterk afgeraden omdat namen handig zijn voor debugging, automatische documentatie, etc.